### ISSUE TITLE
Working?

### ISSUE THREAD

**Issue by zelittle** [jkoehl/homebridge-waterguru#1]
Does the plug-in work in homebridge? I wish I could contribute with code but I can at least test it out starting tomorrow sometime. Getting the water guru delivered tomorrow.

**Comment by jkoehl** [comment]
Not yet. Working on it today actually. It will be a bit rough initially as I figure out how best to integrate it with HomeBridge/Homekit.

**Comment by zelittle** [comment]
Sounds good. Let me know if I can help.

**Comment by kevinjgibb** [comment]
Any luck getting this to work yet?

**Comment by jkoehl** [comment]
I had stopped working on it because I had hoped it reported the water temp search time it called up to the cloud but it seems like it only records the temp when it does the daily water test.

I also found that HomeKit doesn’t have a great way to represent the water chemistry values other than in a custom attribute so it gets a bit buried.

so basically I was trying to convince myself why HomeKit integration will improve upon the WaterGuru app itself. I think I can get it working though I just wasn’t sure anyone would use it.

**Comment by kevinjgibb** [comment]
That is a bit disappointing.  I was hoping to see the actual water temp, especially now that the app shows all of the movement, along with the current temp.  I especially wanted to be able to ask Siri what the conditions in the pool were, even if I had to ask temp specifically, in order to get the most current.  I was envisioning tiles for each specific chemistry reading, but it sounds like that isn't feasible.  The WaterGuru app is much better since the most recent upgrade, I just prefer to have as much info as I can in the Home app.

**Comment by jkoehl** [comment]
Yeah maybe I will take another look to see if they do somehow report current temp. I know it calls home every so often and it seems trivial for it to send the current temp at that time.

But yeah for me I really wanted to know current water temp as the most useful feature to expose to HomeKit but much less so if it is only measured once per day.

At the very least I can try to get a version published that reports whatever temp and info it has and see if it is useful.

**Comment by kevinjgibb** [comment]
I appreciate you taking another look at it!

**Comment by jkoehl** [comment]
Oh hey I just realized the latest app shows hourly temps so it looks like the API should now make it possible for a more live temp.

Yay that may make it worthwhile

**Comment by kevinjgibb** [comment]
Sorry, I just assumed you were on the latest version of the app, hence my prior comment on the movement and current temp.  That improvement alone, made the equipment investment more worthwhile.  I was pumped when reading the update features last week!!

**Comment by kevinjgibb** [comment]
Thank you!! Temp is working great, chlorine and pH don’t seem to appear, but not sure if you intended for them to, in this update.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
